tests/pytests/test_client_server_mtls.py

```diff
@@ -17,43 +17,17 @@
 Basic pytests to exercise stripped-down Client-Server SSL Socket
 communication using mTLS.
 
-# pylint: disable=line-too-long
```

Comment:
Relocated to the bottom of the file, to reduce distraction upon initial reading ...